sharedbuffers/mapped_struct.py

```diff
@@ -20,6 +20,7 @@
 import weakref
 from datetime import timedelta, datetime, date
 from decimal import Decimal
+from math import frexp
```

Comment:
You have to `cython.declare` frexp as object for it to be a "fast" global (otherwise it still goes through the module dict). Or add it to the pxd, both are equivalent.